packages/shared_preferences/shared_preferences/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.5.8
```

Comment:
This will need to be 0.6.0; we learned after the fact with path_provider that the direct registration step makes it a breaking change for certain scenarios (notably, tests).